From c97d13e34c4f3d466cb3404e83a782ca11d3cf41 Mon Sep 17 00:00:00 2001
From: jean charles mammana <x@x>
Date: Thu, 11 Apr 2013 14:25:00 +0800
Subject: [PATCH 54/54] kernel header missing during installation

Some video application require the "usb/video.h" header.
So here the patch to add it during the kernel header installation :
---
 include/linux/usb/Kbuild |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/linux/usb/Kbuild b/include/linux/usb/Kbuild
index ed91fb6..b607f35 100644
--- a/include/linux/usb/Kbuild
+++ b/include/linux/usb/Kbuild
@@ -7,3 +7,4 @@ header-y += gadgetfs.h
 header-y += midi.h
 header-y += g_printer.h
 header-y += tmc.h
+header-y += video.h
-- 
1.7.10.4

